.github/workflows/superset-applitool-cypress.yml

```diff
@@ -36,7 +36,12 @@ jobs:
         ports:
           - 16379:6379
     steps:
-      - uses: actions/checkout@v3
+      - name: "Checkout ${{ github.ref }} ( ${{ github.sha }} )"
+        uses: actions/checkout@v3
```

Comment:
bycatch: we should probably also bump all `actions/checkout@v2` to `actions/checkout@v3` (also in that same follow-up).